src/gateway/kong-enterprise/index.md

```diff
@@ -4,89 +4,97 @@ badge: enterprise
+The [Analytics](/gateway/{{page.kong_version}}/kong-enterprise/analytics/) platform gives you deep insights into services, routes, and application usage data. You can keep your finger on the pulse of the health of your API products with custom reports and contextual dashboards, and you can enhance the native monitoring and analytics capabilities with {{site.base_gateway}} plugins that enable streaming monitoring metrics to third-party analytics providers, such as [Datadog](/hub/kong-inc/datadog/) and [Prometheus](/hub/kong-inc/prometheus/).
 
 [Start monitoring with Vitals &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/analytics/)
 
+## Role-based access control (RBAC)
+
+Kong Enterprise allows the use of built-in RBAC so you can streamline developer onboarding, and create apply fine-grained security and traffic policies using the Admin API, Kong Manager, and the CLI.
+You can configure {{site.base_gateway}} users, roles, and permissions with role-based access control (RBAC), either through the [Admin API](/gateway/{{page.kong_version}}/admin-api/rbac/reference/) or with [Kong Manager](/gateway/{{page.kong_version}}/kong-manager/auth/rbac).
+
+[Manage teams with RBAC &rarr;](/gateway/{{page.kong_version}}/kong-manager/auth/rbac)
+
 ## Secrets management
 
-Application secrets include sensitive data like passwords, keys, certifications, tokens, and other items
-which must be secured. {{site.base_gateway}} supports
-[secrets management](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/), 
-which allows you to store secrets in a vault to help you protect them from accidental exposure. By storing sensitive values as secrets, you ensure that they are not
-visible in plaintext throughout the platform, in places such as `kong.conf`,
-in declarative configuration files, logs, or in the Kong Manager UI.
+{{site.base_gateway}} Enterprise offers out of the box secrets management with the following backends: 
 
-[Secure your application secrets &rarr;](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/)
+* [Amazon Web Services](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/backends/aws-sm/)
+* [Google Cloud Platform](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/backends/gcp-sm/)
+* [Hashicorp Vault](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/backends/hashicorp-vault/)
 
-## Dynamic plugin ordering
+Configuration is simple, {{site.base_gateway}} consumes your key for the backend provider, authenticates with the backend provider, and starts using the backend to centrally manage and store application secrets, sensitive data, passwords, keys, certifications, tokens, and other items.
```

Comment:
```suggestion
{{site.base_gateway}} consumes your key for the backend provider, authenticates with the backend provider, and starts using the backend to centrally manage and store application secrets, sensitive data, passwords, keys, certifications, tokens, and other items.
```
Removing "configuration is simple" because 1, this sentence doesn't talk about configuration, and 2, not great language to use in docs.